controllers/results/operation_results_test.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package results
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Operation Results", func() {
+	var result *OperationResult
+
+	BeforeEach(func() {
+		result = &OperationResult{}
+		Expect(reflect.TypeOf(result)).To(Equal(reflect.TypeOf(&OperationResult{})))
+	})
+
+	It("returns an tuple instructing the reconcile loop to continue processing operations", func() {
+		Expect(ContinueProcessing()).Should(And(
```

Comment:
should we also check that the error is `nil`?

Reply:
fixed.